### ISSUE TITLE
bootstrap_test.TestBootstrap.simple_bootstrap_test

### ISSUE THREAD

**Issue by ptnapoleon** [riptano/cassandra-dtest#182]
https://issues.apache.org/jira/browse/CASSANDRA-8322

**Comment by mambocab** [comment]
I've pulled out a `get_size` function that uses `nodetool info` to pull out the load. It's here:

https://github.com/mambocab/cassandra-dtest/blob/bootstrap-size/bootstrap_test.py#L9

I can move this to `tools.py` if it'd be useful elsewhere, but I'd want to spend a little time making it more robust. Right now it assumes all units are KB, and enforces that with an assert so it doesn't silently say that 2.1 GB is the same as 2.1 KB.

I've seen this fail under 2.1.2 and 2.1.3 with the relationship between the initial and final load on `node1` being wrong, though not by much:

```
FAIL: simple_bootstrap_test (bootstrap_test.TestBootstrap)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mambocab/cstar_src/cassandra-dtest/bootstrap_test.py", line 74, in simple_bootstrap_test
    assert_almost_equal(initial_size, 2 * size1)
  File "/home/mambocab/cstar_src/cassandra-dtest/assertions.py", line 56, in assert_almost_equal
    assert vmin > vmax * (1.0 - error) or vmin == vmax, "values not within %.2f%% of the max: %s" % (error * 100, args)
AssertionError: values not within 16.00% of the max: (466.78, 576.58)

[...]

dtest: DEBUG: initial_size: 466.78
dtest: DEBUG: size1: 288.29
dtest: DEBUG: size2: 280.68
```

There's also an error happening when calling `patient_cql_connection`, shown here:

https://gist.github.com/anonymous/37008ac2a106823c0721

I don't see this error in `counter_tests.py:TestCounters.simple_increment_test`, which also uses `patient_cql_connection`.

**Comment by ptnapoleon** [comment]
That error means one of two things: The cluster didn't start, or the cluster took so long to start that patient timed out. Worth investigating the issue either way.

It's probably not worth moving this into `tools.py` yet, since no one else would be using it.

I'm okay with the test still failing, getting a better size calculation is the first step to stabilizing this test. We can figure out why it continues to fail once this correction is merged.

**Comment by ptnapoleon** [comment]
Fixed by #209
